include/sdk/gx/GXTransform.h

```diff
@@ -4,12 +4,15 @@
 #include <Common.h>
 
 #include <sdk/mtx/mtx.h>
+#include <sdk/mtx/mtx44.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+void GXSetProjection(const Mtx44 *, int);
```

Comment:
- Second arg is GXProjectionType
- I think you want `Mtx44` instead of `Mtx44 *`?